crates/router/src/configs/defaults.rs

```diff
@@ -944,6 +895,94 @@ impl Default for super::settings::RequiredFields {
                     ),
                 ])),
             ),
+            (
+                enums::PaymentMethod::Crypto,
+                PaymentMethodType(HashMap::from([(
+                    enums::PaymentMethodType::CryptoCurrency,
+                    ConnectorFields {
+                        fields: HashMap::from([(
+                            enums::Connector::Cryptopay,
+                            vec![RequiredFieldInfo {
+                                required_field: "currency".to_string(),
+                                display_name: "currency".to_string(),
+                                field_type: enums::FieldType::DropDown {
+                                    options: vec![
+                                        "BTC".to_string(),
+                                        "LTC".to_string(),
+                                        "ETH".to_string(),
+                                        "XRP".to_string(),
+                                        "XLM".to_string(),
+                                        "BCH".to_string(),
+                                        "ADA".to_string(),
+                                        "SOL".to_string(),
+                                        "SHIB".to_string(),
+                                        "TRX".to_string(),
+                                        "DOGE".to_string(),
+                                        "BNB".to_string(),
+                                        "BUSD".to_string(),
+                                        "USDT".to_string(),
+                                        "USDC".to_string(),
+                                        "DAI".to_string(),
```

Comment:
use the `Currency` Enum type. This will restrict us to the currency that is available in the currency enum.

Reply:
Yes, Changed the field type to Currency